dbflow/src/main/java/com/xmartlabs/base/dbflow/controller/DbFlowController.java

```diff
@@ -1,4 +1,4 @@
-package com.xmartlabs.template.controller;
+package com.xmartlabs.base.dbflow.controller;
```

Comment:
I'm fine with both. If we use "database" and "service", new implementations could use their name (like "dbflow").